src/train/trainer/xgboost_interface.py

```diff
@@ -93,15 +93,10 @@ def get_weight_dict(self, node_type):
                 return
             weight_dict[component] = {
                 "All_Weights": {
-                        "Bias_Weight": 0,
                         "Categorical_Variables": dict(),
                         "Numerical_Variables": {self.features[i]: 
-                                                {"scale": scaler.scale_[i],
-                                                "mean": 0, 
-                                                "variance": 0, 
-                                                "weight": model_in_json 
-                                                }
-                                                for i in range(len(self.features))},
+                                                {"scale": scaler.scale_[i]} for i in range(len(self.features))},
+                        "XGboost_Weights": model_in_json 
```

Comment:
@sunya-ch can you use base64 to encode the json so I can get the model in a string? Here is what I have on the kepler side
https://github.com/sustainable-computing-io/kepler/pull/1052/files#diff-21785ede7072be0ccd66e2b884e0be5b77a77e0ab16aa0d93358d2bdd0b32715

Thank you!